internal/tpl/makefile.go

```diff
@@ -78,7 +78,11 @@ func (data templateData) IsNoSource(moduleName string) (bool, error) {
 	if e != nil {
 		return false, e
 	}
-	return buildops.IfNoSource(module), nil
+	noSource := buildops.IfNoSource(module)
+	if !noSource && module.Path == "" {
+		return false, errors.Errorf(`mandatory "path" property of the "%s" module is missing or empty`, moduleName)
```

Comment:
Also please move the message to a constant

Reply:
moved